### ISSUE TITLE
vue-fela package name

### ISSUE THREAD

**Issue by wagerfield** [dustin-H/vue-fela#2]
Hi Dustin,

I was wondering if you were still actively maintaining this repository and package on npm?

Reason I ask is I have just finished writing a package that sets up the bindings between Vue and Fela.

If you have the time and would like to check it out, I have published it to npm here:

https://www.npmjs.com/package/vue-fela-plugin

The plugin sets up bindings between a Fela Renderer instance that is injected during instantiation of the root Vue application and all child Vue component instances. This allows you to reference the Fela Renderer instance within any component via `this.$fela`.

This follows the design pattern that Vuex, Vue Router and other popular plugins have implemented:

```js
import Vue from 'vue'
import VueFela from 'vue-fela-plugin'
import { createRenderer } from 'fela'

Vue.use(VueFela)

const app = new Vue({
  fela: createRenderer()
}).$mount('#app')
```

The plugin then provides some helpers for mapping Fela rules to computed props (much like your implementation). However the main difference here is that in this implementation, the Vue component instance is passed to each rule so that its properties can be referenced within the rule:

```vue
<template>
  <div :class="box"/>
</template>

<script>
import { mapRule } from 'vue-fela-plugin'

const rule = ({ size, fill }) => ({
  backgroundColor: fill,
  width: size,
  height: size
})

export default {
  props: {
    size: {
      default: '100px'
    },
    fill: {
      default: 'red'
    }
  },
  computed: {
    box: mapRule(rule)
  }
}
</script>
```

Finally, it includes a `<fela/>` provider component that takes care of rendering the cached styles to the DOM on both the client and server simply by including the component once in your application:

```vue
<template>
  <fela id="app">
    <box size="200px" fill="blue"/>
  </fela>
</template>
```

There is a [full working example](https://github.com/wagerfield/vue-fela/tree/master/example) of the plugin in use within a simple Nuxt application if you would like to check that out.

If you are no longer maintaining this repo and package on npm, I would be eternally grateful if you would release the `vue-fela` package to me and in return I would happily accredit you for doing so in the README. I also hope that you find this package useful if you are still using these 2 technologies in your stack 😃 

Many thanks for your consideration ❤️

**Comment by dustin-H** [comment]
Hi @wagerfield,

great job! I haven't had the time to create a full vue-fela binding so I focused on the features I needed.

I'm not using vue anymore. This is why I'm not maintaining this repo.

However, the current version of `vue-fela` needs to stay in npm to not crash the production-build of my customers.

I can give you access to the npm package, then you can release your version of vue-fela as `1.0.0`.

**Comment by dustin-H** [comment]
I just published v1.0.0 to ensure that this package is running for everyone using it as it is at the moment.

You are collaborator of `vue-fela` now. Just publish any version greater than 1.0.0.

Can you give me an information when you published your code. Then I will create a link to your repo on the readme.

**Comment by wagerfield** [comment]
Thanks a million @dustin-H, really appreciate you doing this.

I have just published to `vue-fela@2.0.0` 🎉
